test/lib/ui5framework/npm/Registry.js

```diff
@@ -58,31 +71,23 @@ test.serial("_getPacoteOptions", async (t) => {
 		fake: "config",
 		agent: false
 	};
-
-	libnpmconfigReadToJSON.returns(npmConfig);
+	npmConfigFlat.value(npmConfig);
```

Comment:
... and `value` in here?

Reply:
Thanks! Fixed. Now everywhere is `value`.